lsstvaultutils/__init__.py

```diff
@@ -1,6 +1,6 @@
+from typing import Sequence
 from .tokenadmin import AdminTool
 from .recursivedeleter import RecursiveDeleter
-from .secretcopier import SecretCopier
 from .vaultconfig import VaultConfig
 
-__all__ = [AdminTool, RecursiveDeleter, SecretCopier, VaultConfig]
+__all__: Sequence[str] = ["AdminTool", "RecursiveDeleter", "VaultConfig"]
```

Comment:
Nit: You don't need a type here. mypy will figure out that it's a list of str.

Reply:
Thanks.